pkg/httpproxy/sign.go

```diff
@@ -235,3 +235,16 @@ type basic struct{}
 type digest struct{}
 
 type arbitrary struct{}
+
+func ignoreHeaders(req *http.Request) map[string]string {
+	data := map[string]string{}
+	for header, val := range req.Header {
+		if strings.EqualFold(header, ConnectionHeader) || strings.HasPrefix(strings.ToLower(header), ForwardHeaderPrefix) {
+			if len(val) > 0 {
+				data[header] = strings.Join(val, "")
```

Comment:
why are we joining with an empty string?

Reply:
val is of type `[]string`